.github/workflows/ci.yml

```
@@ -21,10 +21,10 @@ jobs:
     name: Build Docs
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
       with:
-        python-version: '3.10'
+        python-version: '3.11'
```

```suggestion
        python-version: '3.10'
```

I think I prefer testing the docs on 3.10 for the time being. There is definitely value in having 3.11 in the unit tests, but I don't think the docs will get any direct benefit. RTD actually builds them on 3.9, so if anything we may wish to match that and then upgrade here once RTD bumps that default.